app/assets/stylesheets/nav.scss

```diff
@@ -21,36 +48,75 @@ $button-color: #1E90FF;
 
   input {
     width:280px;
+    border-radius: 5%;
   }
+
   label {
       color: orange;
       font-weight: bold;
+
+      span{
+        color: $purple;
+        font-size: 10px;
+      }
+
     }
+}
 
+.block {
+  display: inline-block;
 }
 
 .form {
   margin-left: 10%;
+  display: inline-block;
+}
+
+.centered_bar {
+  margin-top: 10em;
+  margin-left: 10%;
+}
 
+.home_text{
```

Comment:
Selector `home_text` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space